tests/model/ArrayListTest.php

```diff
@@ -746,7 +746,8 @@ public function testCanFilterByEmpty() {
 		$this->assertFalse($list->canFilterBy('Age'));
 	}
 
-	public function testByID() {
+	public function testByID()
+	{
```

Comment:
Unfortunately you can't dictate code style like this, it's more white space based. :(